torchao/quantization/qat/linear.py

```diff
@@ -107,12 +107,23 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return F.linear(x, w)
 
 
+class _LegacyQATQuantizer(TwoStepQuantizer):
```

Comment:
nit: why is this called legacy, I thought we want to use the module swap API in the long term, or are we going to change this API to function call be still use module swap underneath?

Reply:
yeah, in the long term we plan to use `quantize_` for module swap. These quantizers are too tailored to particular quantization schemes